explorer/interpreter/value_kind.def

```diff
@@ -0,0 +1,59 @@
+// Part of the Carbon Language project) under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Note that this is an X-macro header.
+//
+// It does not use `#include` guards) and instead is designed to be `#include`ed
```

Comment:
Looks like some search replace `,` -> `)` went too far

Reply:
> Looks like some search replace `,` -> `)` went too far

Whoops, fixed.

Btw, is there a way to make git[hub] indicate that this file is a copy of value.h to better preserve history?